packages/rspack/src/index.ts

```diff
@@ -1,3 +1,5 @@
+import type Watching from "./watching";
+import type { DevServer } from "./config";
```

Comment:
This is not exported by webpack. Shouldn't we align this?

Reply:
yes，actually webpack can access this type by other way and they just dont export it at top level

Reply:
Webpack does not directly export this, if you are trying to access the configuration of `devServer`, you can just follow `Configuration['devServer']`

Reply:
lets align with webpack first

Reply:
Okay, let's remove `DevServer` then